include/hx/GC.h

```diff
@@ -359,7 +356,29 @@ class ImmixAllocator
    // These allocate the function using the garbage-colleced malloc
    inline static void *alloc(ImmixAllocator *alloc, size_t inSize, bool inContainer, const char *inName )
    {
+
+   	  if (!inContainer) {
+   	  	
+   	  	// a non-object with 0 bytes length ? it would make the alloc memory type
+   	  	// check mask in mistake
+   	  	if (inSize == 0)
+   	  		 return 0;
+   	  		
+   	  	inSize = (inSize+3)&~3;
+   	  }
+   	  else
+   	  {
+	      #if defined(HXCPP_VISIT_ALLOCS) && defined(HXCPP_M64)
+	      // Make sure we can fit a relocation pointer
+	      inSize += 4;
+	      #endif
+   	  }
+   	  	
+   	 // if (inContainer == false && inSize > IMMIX_LARGE_OBJ_SIZE)
+   	  	
+   	    	 
       #ifdef HXCPP_GC_NURSERY
+      

```

Comment:
Please try to be consistent with the indentation here, I believe hxcpp uses 3 spaces for a tab. (Don't ask me why…)

Reply:
You are right, I'm using UEDIT32 and I think that hughsando is using MSVC.

Reply:
Plus // if (inContainer == false && inSize > IMMIX_LARGE_OBJ_SIZE) this should be removed